Site/ASAB Maestro/Descriptors/mongo.yaml

```diff
@@ -9,26 +9,19 @@ descriptor:
   volumes:
     - "{{SLOW_STORAGE}}/{{INSTANCE_ID}}/data:/data/db"
     - "{{SITE}}/{{INSTANCE_ID}}/conf:/etc/mongo:ro"
-    - "{{SITE}}/{{INSTANCE_ID}}/script:/script:ro"
+    - "{{SITE}}/{{INSTANCE_ID}}/mongo:/script:ro"
```

Comment:
I recommend to keep this aligned: 
`"{{SITE}}/{{INSTANCE_ID}}/script:/script:ro"`

... to maintain at least bit of sanity ;-)

Reply:
`{{INSTANCE_ID}}` is `mongo-1` e.g.

Reply:
in this very repository

Reply:
Okej. I guess I got that ... but you don't need to put the script into the library.
You can build it directly to the "site" folder.